crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1730,6 +1730,37 @@ WHERE e1.epoch = e2.epoch
             .as_str(),
         )
     }
+
+    async fn get_all_epoch_address_stats(
```

Comment:
can we process this stat by the end of this epoch? It's expensive to run this on the fly  for every checkpoint, especially when the results are static

Reply:
the address stats is calculated in an async processor, similar to post-processing and now calculated each 100 checkpoints.
Product folks want freshness better than epoch but we can easily adjust the number of checkpoints if needed.